tests/select/test_select_pv_systems_without_output_wholeday.py

```diff
@@ -4,42 +4,33 @@
 import pandas as pd
 import xarray as xr
 
-from ocf_datapipes.select import DropLessThanOneDay, RemoveBadSystems
+from ocf_datapipes.select import TrimDatesWithInsufficentData, RemoveBadSystems
 from ocf_datapipes.select import SelectSysWithoutOutputWholeday as SysWithoutPV
 
 
 # TODO Still needed to work on this tests a bit
-def test_sys_with_lessthan_oneday_pv():
```

Comment:
I would leave in this constructed example as well, and just add a different test with the passive one. We want it to pass both.